### DISCUSSION TITLE
Any updates on a release to resolve  CVE-2024-1135?

### DISCUSSION THREAD

**Discussion by swils23** [benoitc/gunicorn#3186] [General]
... but I will make a release of Gunicorn this week .

_Originally posted by @benoitc in https://github.com/benoitc/gunicorn/issues/3162#issuecomment-1966890066_
            
https://github.com/advisories/GHSA-w3h3-4rj7-4ph4

**Comment by swils23** [comment]
It looks like this was fixed in [this](https://github.com/benoitc/gunicorn/commit/ac29c9b0a758d21f1e0fb3b3457239e523fa9f1d) commit on master in December 2023, but the latest release was back in July 2023.

**Reply by benoitc** [reply]
Thanks for opening this topic. Gunicorn 22.0.0 has been released. This is fixing this issue.
